designSystem/build.gradle.kts

```
@@ -34,7 +34,7 @@ android {
 }
 
 dependencies {
-
+    implementation(project(":app"))
```

Hi @monaayman123, this is what cause the circular dependency error and let the workflow fail please remove this line. The app already implements the designSystem module and there's no need to be the opposite as well

---

font not defined if i removed it

---

@monaayman123 can you explain the problem please?